frontend/.env

```
@@ -0,0 +1 @@
+VITE_CLERK_PUBLISHABLE_KEY=pk_test_cG9saXRlLWJlbmdhbC05My5jbGVyay5hY2NvdW50cy5kZXYk  
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)